app/helpers/view_components/track/exercise_header.rb

```diff
@@ -104,7 +104,7 @@ def mentoring_tab
       end
 
       def show_iterations_tab? = !user_track.external?
-      def show_approaches_tab? = !exercise.tutorial? && !!user&.can_view_approaches?
+      def show_approaches_tab? = !exercise.tutorial? && exercise.has_approaches?
```

Comment:
Does this mean that merging this tab effectively puts things live?

Reply:
It would. I had expect me to finish the individual approaches stuff, but unfortunately I've not yet been able to do that. I'm happy to revert.